packages/eas-cli/src/commands/branch/publish.ts

```diff
@@ -1,51 +1,100 @@
 import { getConfig } from '@expo/config';
 import { Command, flags } from '@oclif/command';
+import { uniqBy } from '@oclif/plugin-help/lib/util';
+import assert from 'assert';
 import chalk from 'chalk';
 import Table from 'cli-table3';
+import dateFormat from 'dateformat';
+import gql from 'graphql-tag';
 import ora from 'ora';
 
+import { graphqlClient, withErrorHandlingAsync } from '../../graphql/client';
+import {
+  GetUpdateGroupAsyncQuery,
+  Maybe,
+  Robot,
+  RootQueryUpdatesByGroupArgs,
+  Update,
+  UpdateInfoGroup,
+  User,
+} from '../../graphql/generated';
 import { PublishMutation } from '../../graphql/mutations/PublishMutation';
 import Log from '../../log';
 import { ensureProjectExistsAsync } from '../../project/ensureProjectExists';
-import {
-  findProjectRootAsync,
-  getBranchByNameAsync,
-  getProjectAccountNameAsync,
-} from '../../project/projectUtils';
+import { findProjectRootAsync, getProjectAccountNameAsync } from '../../project/projectUtils';
 import {
   buildBundlesAsync,
   buildUpdateInfoGroupAsync,
   collectAssets,
   uploadAssetsAsync,
 } from '../../project/publish';
-import { promptAsync } from '../../prompts';
+import { promptAsync, selectAsync } from '../../prompts';
 import { getLastCommitMessageAsync } from '../../utils/git';
+import { viewUpdateBranchAsync } from './view';
+
+type PublishPlatforms = 'android' | 'ios';
 
+export async function getUpdateGroupAsync({
+  group,
+}: RootQueryUpdatesByGroupArgs): Promise<
+  Pick<Update, 'group' | 'runtimeVersion' | 'manifestFragment' | 'platform' | 'message'>[]
+> {
```

Comment:
```suggestion
async function getUpdateGroupAsync({
  group,
}: RootQueryUpdatesByGroupArgs): GetUpdateGroupAsyncQuery
['updatesByGroup'] {
```

Reply:
It would be really convenient, but I don't think we can use types like this: `GetUpdateGroupAsyncQuery['updatesByGroup']`